Jenkinsfile

```diff
@@ -65,9 +65,9 @@ pipeline {
                         }
                     }
                 }
-                stage("linux-newer-core") {
+                stage("linux-newer-jenkins-core") {
                     agent {
-                        label "highmem"
+                        label "linux"
```

Comment:
Why change off `highmem`? The `linux` agents are smaller, and Declarative's tests can end up timing out on them.

Reply:
Trying to use the standard machine config if possible.  I'll revert.

Reply:
Thanks!